src/main/kotlin/uk/gov/dluhc/printapi/service/PrintResponseProcessingService.kt

```diff
@@ -18,11 +20,16 @@ class PrintResponseProcessingService(
     private val printDetailsRepository: PrintDetailsRepository,
     private val idFactory: IdFactory,
     private val clock: Clock,
-    private val statusMapper: StatusMapper
+    private val statusMapper: StatusMapper,
+    private val processPrintResponseMessageMapper: ProcessPrintResponseMessageMapper,
+    private val processPrintResponseQueue: MessageQueue<ProcessPrintResponseMessage>
 ) {
 
     fun processBatchAndPrintResponses(printResponses: PrintResponses) {
-        // TODO: EIP1-2420
+        processBatchResponses(printResponses.batchResponses)
+        printResponses.printResponses.forEach {
```

Comment:
`printResponses.printResponses` - generated classes from our `PrintResponses.yaml` spec... guess nothing we can do about this, but slightly frustrating.